### ISSUE TITLE
Bitwig beta 11 bricks theming

### ISSUE THREAD

**Issue by DerpcatMusic** [Berikai/bitwig-theme-editor#12]
When changing theme (through powershell, administrator) 
doesnt work and shows this

```
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException: Cannot read field "instructions" because "this.methodNode" is null
        at dev.berikai.BitwigTheme.core.advanced.components.TimelinePlayheadClass.setTheme(TimelinePlayheadClass.java:54)
        at dev.berikai.BitwigTheme.core.advanced.AdvancedThemeManagerClass.setTheme(AdvancedThemeManagerClass.java:23)
        at dev.berikai.BitwigTheme.Main.applyTheme(Main.java:99)
        at dev.berikai.BitwigTheme.UI.MainUI.lambda$new$0(MainUI.java:30)
        at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
        at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2313)
        at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
        at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
        at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:279)
        at java.desktop/java.awt.Component.processMouseEvent(Component.java:6616)
        at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3398)
        at java.desktop/java.awt.Component.processEvent(Component.java:6381)
        at java.desktop/java.awt.Container.processEvent(Container.java:2266)
        at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4991)
        at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
        at java.desktop/java.awt.Component.dispatchEvent(Component.java:4823)
        at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4948)
        at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4575)
        at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4516)
        at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
        at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2780)
        at java.desktop/java.awt.Component.dispatchEvent(Component.java:4823)
        at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:775)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
        at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:747)
        at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
        at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:744)
        at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
        at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by Berikai** [comment]
Hi Derpcat! Know you since the Hypocrisy track, love your music. Keep it up!

Due to a breaking change Bitwig made in version 5.2 Beta 11 about the Timeline Playhead, version 1.4.0 is giving error while trying to change Timeline Playhead.

For now, the workarounds for this issue are either using [Bitwig Theme Editor 1.3.1](https://github.com/Berikai/bitwig-theme-editor/releases/tag/1.3.1) or removing advanced object part from the theme json file. I already created an [issue](https://github.com/Berikai/bitwig-theme-editor/issues/11) regarding indicating that situation.

**Comment by Berikai** [comment]
This has been fixed with the latest release and can be closed now: [Release 1.4.1](https://github.com/Berikai/bitwig-theme-editor/releases/tag/1.4.1)
